.environment/gitleaks/run-gitleaks.sh

```diff
@@ -91,7 +93,7 @@ function scan_uncommitted() {
     note "Scanning your suggested changes."
     # NOTE: ironically, the switch to scan your staged (i.e. to be committed) changes is to use the --unstaged switch
 
-    base_command "--unstaged"
+    base_command "--staged"
```

Comment:
Is the comment above "# NOTE: ironically, the switch to scan your staged (i.e. to be committed) changes is to use the --unstaged switch" no longer true?

Reply:
Yes, update comment.